language/reserved_keywords.rb

```diff
@@ -0,0 +1,159 @@
+require_relative '../spec_helper'
+
+describe "Ruby's reserved keywords" do
+  # Copied from Prism::Translation::Ripper
```

Comment:
Ripper is not really the source, let's link to https://github.com/ruby/ruby/blob/master/defs/keywords which is closer to be the true source.
We could also link to some Prism `.c` file instead if there is a nice list of keyword there, as that would be the true-est source.

Reply:
Right, let's just add a comment with a link to https://github.com/ruby/ruby/blob/master/defs/keywords then.

> It would be nice if there was a `Prism.keywords` method that would just hand us the list.

We wouldn't want to need to require the prism gem in specs anyway (notably it's not bundled with older Rubies).

Reply:
I meant this as an additional validation in the CI, so if Ruby 3.5 adds a new keyword to the grammar, we'll get a notification that this local copy is out of date.